Automattic/wp-calypso
# Drop support for package-relative imports

Closes #44602

#### Changes proposed in this Pull Request

* Remove eslint rule `wpcalypso/no-package-relative-imports`
* Drop support for package-relative imports in Webpack config (client, server and desktop)
* Drop support for package-relative imports in Jest config (client, integration and server)

#### Testing instructions

* ✅  ~~Verify all tests are green. This means Calypso can compile~~
* ✅  ~~Verify ICFY reports no change in bundle size (or a very small one).~~
* Smoke tests the live branch
* Smoke test the desktop app
